src/client/NetworkingClient.cpp

```diff
@@ -95,27 +97,25 @@ void NetworkingClient::RealtimeClientRecv() {
     sf::Uint32 client_id;
     float x_pos;
     float y_pos;
+    float size;
 
     while (true) {
         sf::Packet packet;
         sf::IpAddress sender;
         unsigned short port;
         mRealtimeClient->receive(packet, sender, port);
-        std::cout << "recv: " << sender << " " << port << std::endl;
         // fetch state updates for now
         if (packet >> server_tick) {
-            mAcceptingPositionRead = false;
-            mPositions.clear();
-            while (packet >> client_id >> x_pos >> y_pos) {
-                std::cout << "Client: " << client_id << " has position x,y: " << x_pos << " " << y_pos << std::endl;
-                position p = {client_id, x_pos, y_pos};
-                mPositions.push_back(p);
+            mAcceptingNetworkGameObjectsRead = false;
+            mNetworkGameObjects.clear();
+            while (packet >> client_id >> x_pos >> y_pos >> size) {
```

Comment:
shouldn't this be `packet >> ngo`?

Reply:
To achieve this, I think we need extend packets to support your custom types: https://www.sfml-dev.org/tutorials/2.5/network-packet.php
Didn't feel like doing that in this branch.